src/sbr/easinTemplates.py

```diff
@@ -53,6 +53,7 @@
     hibernate:
       ddl-auto: update
     show-sql: true
+    properties.hibernate.temp.use_jdbc_metadata_defaults: false
```

Comment:
this is wrong , you should use yaml conf mode :

```suggestion
    hibernate:
      temp:
        use_jdbc_metadata_defaults: false

```

Reply:
you did not add this ?